config/initializers/clearance.rb

```diff
@@ -0,0 +1,5 @@
+Clearance.configure do |config|
+  config.routes = false
+  config.mailer_sender = ENV['TEST_SENDER_EMAIL']
```

Comment:
This would be causing the `SMTP From address may not be blank: nil` error in tests.  To fix it you could do a fetch and set a default or only set if in production. Or maybe just set it if the environment variable is present. I think I like the 3rd option because it will use the clearance default if not set and you can set it but is not required in any environment.

```
config.mailer_sender = ENV.fetch('MAIL_SENDER', 'reply@example.com')
config.mailer_sender = ENV['MAIL_SENDER'] if Rails.env.production?
config.mailer_sender = ENV['MAIL_SENDER'] if ENV['MAIL_SENDER'].present?
```

Reply:
Is 'MAIL_SENDER' a default name. Or should I change the env to just say mail sender instead of test?